eureka/src/main/java/com/linecorp/armeria/server/eureka/EurekaUpdatingListener.java

```diff
@@ -202,6 +201,10 @@ private InstanceInfo fillAndCreateNewInfo(InstanceInfo oldInfo, Server server) {
         final PortWrapper oldSecurePortWrapper = oldInfo.getSecurePort();
         final PortWrapper securePortWrapper = portWrapper(server, oldSecurePortWrapper, SessionProtocol.HTTPS);
 
+        final String instanceId =
+                oldInfo.getInstanceId() != null ? oldInfo.getInstanceId()
+                                                : hostName + ":" + appName + ":" + portWrapper.getPort();
```

Comment:
Could we fall back to `securePortWrapper.getPort()` if `portWrapper` is disabled?

Reply:
Gentle ping. @eisig

Reply:
@ikhoon updated now.